core/variant/variant_setget.cpp

```diff
@@ -1833,6 +1833,7 @@ Variant Variant::iter_get(const Variant &r_iter, bool &r_valid) const {
 			int idx = r_iter;
 #ifdef DEBUG_ENABLED
 			if (idx < 0 || idx >= arr->size()) {
+				_err_print_index_error(FUNCTION_STR, __FILE__, __LINE__, idx, arr->size(), _STR(m_index), _STR(m_size));
```

Comment:
`_err_print_index_error` isn't really meant to be used outside `ERR_FAIL*_INDEX*` macros.

I understand why you used it here as the macro can't be used, but IMO calling `ERR_PRINT` with a custom message might be cleaner.